aws/v8.2.0/release.yaml

```diff
@@ -26,3 +26,4 @@ spec:
     version: 1.14.3
   date: "2019-06-03T10:00:00Z"
   state: deprecated
+  endOfLifeDate: "2019-09-01T00:00:00Z"
```

Comment:
This was last year. What does that mean now for the customers?

Reply:
Ouch. Mistake.